compiler/src/parsing/parser.messages

```diff
@@ -5933,7 +5927,7 @@ program: WASMI64 COLON LPAREN UIDENT COMMA WHILE
 ## typ COMMA
 ##
 
-Expected a type or an immediate `)` to complete the tuple type.
+Expected another type in the tuple type.
```

Comment:
Would just move this to the generic `Expected a type` now since the special case to create a tuple type is gone, since if you type `(Number,` it's not yet clear whether you're writing a tuple type or going to go on and write a function type. Unfortunately it's not really obvious from the parser because the lexer injects special `FUN` tokens for functions, but it won't have done that unless you wrote the arrow already.

I guess we can write comments in the messages file, but I fear they'll get lost in the massive sea that's already there 😛 

That plus running `npm run compiler parser:update-errors` would also obliterate them.